app/routes/events/components/EventDetail/infoLists.tsx

```diff
@@ -105,6 +113,66 @@ export const useDeadlineInfoList = (event?: DetailedEvent) => {
           ),
         }
       : null,
+    activationTimeMoment.isBefore(currentMoment) &&
+    !sameRegistrationAndUnregistration
+      ? {
+          key: 'Frist for påmelding',
+          keyNode: (
+            <TextWithIcon
+              iconNode={<CircleHelp />}
+              content={
+                currentMoment.isBefore(registrationCloseTimeMoment)
+                  ? 'Påmelding stenger'
+                  : 'Påmelding stengte'
+              }
+              tooltipContentIcon={
+                <>
+                  Etter påmeldingen stenger er det ikke mulig å melde seg på
+                  arrangementet
+                </>
+              }
+              iconRight
+              size={14}
+            />
+          ),
+          value: (
+            <FormatTime
+              format="dd DD. MMM HH:mm"
+              time={registrationCloseTimeMoment}
+            />
+          ),
+        }
+      : null,
+    activationTimeMoment.isBefore(currentMoment) &&
+    !sameRegistrationAndUnregistration
+      ? {
+          key: 'Frist for avmelding',
+          keyNode: (
+            <TextWithIcon
+              iconNode={<CircleHelp />}
+              content={
+                currentMoment.isBefore(unregistrationCloseTimeMoment)
+                  ? 'Avmelding stenger'
+                  : 'Avmelding stengte'
+              }
+              tooltipContentIcon={
+                <>
+                  Etter avmeldingen stenger er det ikke mulig å melde seg av
+                  arrangementet
+                </>
+              }
```

Comment:
what's happening here? Why is it called [..]Icon when it takes in text?

Reply:
Should probably be renamed to `iconTooltipContent` or something as I'm assuming it's the content for the tooltip appearing over the icon

Reply:
Agree the `[..]Icon` does not fit there. Renamed to `tooltipContent` as the type indicated, there is only one tooltip in the component so should be good enough name.